templates/vue/src/components/Lightbox/index.vue

```diff
@@ -24,6 +25,7 @@
       :node="node"
       :rowRefs="rowRefs"
       :dimensions="dimensions"
+      @handle-page-menu-click="handlePageMenuClick"
     />
```

Comment:
Can we move this into `multi-content-media` as it's only applicable when that's the content type? I want to keep Lightbox uncluttered if possible.